### PR TITLE
Add star and fork count on library page.

### PR DESCRIPTION
**Pull request by Piicksarn**
- On each library's page, the PR use [github-buttons
  ](https://github.com/ntkme/github-buttons)for showing the **Star** and **Fork** of each git repository.
  - Modified `library.html`.
- To show the Star and Fork of git repopository, it required **`user`** and **`repository`** of git url.
  - Use method in **`reindex.js`** _(from line 78 - 105)_ for searching repository information.
  - Modified `webServer.js`

### CODE REVIEW

File: webServer.js [hunk 1]
```diff
@@ -160,6 +160,40 @@ function start() {
...
+         if (!m) {
+           return null;
+         }
+         return { user: m[1], repo: m[2].replace(/.git$/, '') };
+       }));
+  
+       library.repo = '';
+       if (repos.length > 0) {
+         var repo = repos[0]; // fetch only the first repository
+
```
Comment (PeterDaveHello) [inline webServer.js]: what's this empty for?

**Comment by PeterDaveHello** [comment]
Please give us a preview screenshot.

File: webServer.js (hunk 1 above)
Comment (Piicksarn) [inline webServer.js]: sorry...it just nothing..

**Comment by Piicksarn** [comment]
@PeterDaveHello, the screenshot is shown below:)

star

**Comment by PeterDaveHello** [comment]
I think you should validate url first not collect them first, and you don't even make sure url field exist, and I wonder if we can just use static data, like storing star and fork count in packages.min.json, and just update it in every reindexing, so that it'll be faster.

**Comment by Piicksarn** [comment]
ok, i will keep trying :facepunch:

**Comment by PeterDaveHello** [comment]
ping

**Comment by PeterDaveHello** [comment]
Done in #60

**Pull request closed without merging**